servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp

```diff
@@ -621,7 +621,7 @@ void SceneShaderForwardClustered::init(RendererStorageRD *p_storage, const Strin
 
 		//builtins
 
-		actions.renames["TIME"] = "scene_data_block.data.time";
+		actions.renames["TIME"] = "global_time";
```

Comment:
I believe these are global (affect both vertex and fragment), but you only added this to the `#[vertex]` section.

My recommendation for `#[fragment]` is to add:
```
#define global_time scene_data_block.data.time
```
Which should preserve the same behavior without having to declare and initialize the global variable there.